src/Uno.Extensions.Localization.UI/LocalizationConfiguration.cs

```diff
@@ -1,6 +1,13 @@
 ﻿namespace Uno.Extensions.Localization;
 
+/// <summary>
+/// Contains configuration for the localization feature.
+/// </summary>
 public record LocalizationConfiguration
```

Comment:
We might be able to make LocalizationConfiguration internal, since we now wrap it using the LocalizationService

Reply:
Addressed with 75f9665a96b59f6b97c160a99450674ab620a73a

Reply:
@nickrandolph I needed to keep `LocalizationConfiguration` public for now because test harness needs to instantiate that type when loading l10n info on app startup. 

We could make a change later on: either to the test harness project, or consider adding an extension which includes a set of provided culture info strings. This obviously is not to encourage hardcoding such values instead of a configuration source like `appsettings.json`, but it would simplify how culture infos are programmatically included by their `string`.